prow/config.yaml

```diff
@@ -78,6 +78,38 @@ presubmits:
       - name: cache-ssd
         hostPath:
           path: /mnt/disks/ssd0
+  kubernetes/dns:
+  - name: integration-test
+    agent: kubernetes
+    context: integration-test
+    branches:
+    - master
+    rerun_command: "/test integration-test"
+    trigger: "(?m)^/test( all| pull-community-verify),?(\\s+|$)"
+    always_run: true
+    spec:
+      containers:
+      - image: gcr.io/google_containers/busybox
+        args:
+        - "/bin/sh"
+        - "-c"
+        - "echo 'this is a placeholder'"
+  kubernetes/ingress-gce:
+  - name: integration-test
+    agent: kubernetes
+    context: integration-test
+    branches:
+    - master
+    rerun_command: "/test integration-test"
```

Comment:
s/pull-community-verify/integration-test/
Also this should probably have a different name from the DNS one?
Something like `pull-dns-integration-test` and `pull-ingress-gce-integration-test`